src/navigation/components/Logo/Logo.tsx

```diff
@@ -14,13 +14,16 @@ const b = block('logo');
 
 export type LogoProps = ThemedNavigationLogoData & {
     className?: string;
+    alt?: string;
 };
 
-const Logo: React.FC<LogoProps> = (props) => {
+const Logo: React.FC<LogoProps> = ({alt = 'Logo', ...restProps}) => {
```

Comment:
we use `i18`, and I think we need translation for alt in this place, you can see example in `src/components/Security`

Reply:
Added translation